crates/proto/src/lib.rs

```diff
@@ -4,5 +4,12 @@ use serde::{Deserialize, Serialize};
 pub struct Rustacean {
     pub name: String,
     pub image: String,
+    pub bio:String,
     pub github_url: String,
+    pub email:String,
+    pub geographical_location: String,
+    pub personal_website:Option<String>,
+    pub mastodon_fediverse:Option<String>, 
+    pub IRC:Option<String>,
```

Comment:
The IRC chat should have two properties:

- The user's IRC ID
- User's channels

I think we could have the struct:

```struct
struct Irc {
  id: String,
  channels: HashSet<String>,
}
```